installer.sh.in

```diff
@@ -1386,6 +1584,8 @@ menu() {
             "RootPassword" "Set system root password" \
             "UserAccount" "Set primary user name and password" \
             "BootLoader" "Set disk to install bootloader" \
+            "Lvm" "Set use of LVM" \
+            "Encryption" "Set encrypted partition" \
```

Comment:
I feel like these steps should be optional and say they are optional.

The solution will be overkill and possibly wrong for most users.

Reply:
Could be.
But the default is, not to use LVM and LUKS, so you can easily click.

What is the best way, make this optional? Form my point, you get more steps to finish the setup.

Reply:
Just putting the text "(Optional)" for the menu or section headings

Reply:
How about Optional for LVM and Recommended for LUKS? It makes it clear that they are not mandatory, but it encourages the use of encryption.

Reply:
I think, the installer shoud bring the topics on the table, so the user can choose or not. 

The topic's has many facets and experienced users won't chosse the installer. So no problem to only tag LVM Optional.